.helm/deploy-to-cluster.sh

```diff
@@ -98,11 +99,13 @@ for i in 1; do
     values="$values --set $imagespec.image.repository=docker.io/${docker_hub_account}/ecamp3-api-$imagespec"
   done
 
+  values="$values --set apiCache.image.repository=docker.io/${docker_hub_account}/ecamp3-varnish"
+
   values="$values --set postgresql.dbBackupRestoreImage.pullPolicy=$pull_policy"
   values="$values --set postgresql.dbBackupRestoreImage.repository=docker.io/${docker_hub_account}/ecamp3-db-backup-restore"
 
   helm uninstall ecamp3-"$instance_name"-"$i" || true
-  helm upgrade --install ecamp3-"$instance_name"-"$i" $SCRIPT_DIR/ecamp3 $values
+  helm upgrade --install --debug --dry-run ecamp3-"$instance_name"-"$i" $SCRIPT_DIR/ecamp3 $values
```

Comment:
this should be reverted again

Reply:
reverted with https://github.com/ecamp/ecamp3/pull/3610/commits/a532c219717b0e91b602a68700c4bae99b751051